catalog/MDCCatalog/MDCThemePickerViewController.swift

```diff
@@ -27,6 +27,17 @@ private func createSchemeWithPalette(_ palette: MDCPalette) -> MDCSemanticColorS
   return scheme
 }
 
+class MDCColorThemeCellConfiguration {
+  var name: String
```

Comment:
If these are immutable they should be `let`s

Reply:
Done.

Reply:
they still look like `var`

Reply:
This is also a struct potential

Reply:
Sorry, committed but didn't push it up.